src/Layer/PointCloudLayer.js

```diff
@@ -90,6 +90,18 @@ function changeIntensityRange(layer) {
     }
 }
 
+function changeElevationRange(layer) {
+    if (layer.material.elevationRange) {
+        layer.material.elevationRange.set(layer.minElevationRange, layer.maxElevationRange);
+    }
```

Comment:
- The properties `minElevationRange` and `maxElevationRange` shall be documented.
- Add a guard when `minElevationRange` == `maxElevationRange` to prevent divide by zero
- Use the `?.` operator:
```suggestion
    layer.material.elevationRange?.set(layer.minElevationRange, layer.maxElevationRange);
```